.env.template

```
@@ -0,0 +1,13 @@
+KEYSTORE_PASSWORD=my_keystore_password
+STARKNET_ACCOUNT=path/to/account.json 
+STARKNET_KEYSTORE=/path/to/keystore.json
+DEPLOYER_ADDRESS=="0x to your account"
+
+
+#mainet/testnet/devnet
+STARKNET_RPC='http://0.0.0.0:5050'
+
+#Nethermin juno rpc works perfectly but you can try another one ( not every rpc works)
+#Here it's sepolia but can be main
+FORK_URL='https://rpc.nethermind.io/sepolia-juno/?apikey=<YOUR_API_KEY>'
```

```suggestion
#Here is Sepolia URL, you can replace by a mainnet one.
FORK_URL='https://free-rpc.nethermind.io/sepolia-juno/'
```

Replaced with an URL that doesn't need API_KEY